### ISSUE TITLE
Feature Request: Allow to define an input folder

### ISSUE THREAD

**Issue by iagobruno** [remoteinterview/zero#105]
## Current Behavior

Zero pick up files and folders in the project root folder, this is ok for small projects but can get messy for sites with many pages. Look at this:

![image](https://user-images.githubusercontent.com/3616259/70363191-2c77c900-1866-11ea-8051-f86286591c06.png)

## Proposal

Allow to define an input folder where Zero will get the files, for example:

```js
zero src
zero pages/public // or
zero ../client/pages // or
```

And so the spread files can be organized into a "src" folder like that:

![image](https://user-images.githubusercontent.com/3616259/70363311-aa3bd480-1866-11ea-9fb3-fe0f6a7efb64.png)

This also solves the confusion in .zeroignore to route only the desired files.

## Backward compatibility

This new behavior can be easily integrated without breaking old projects. If no entry point is specified, it will fallback to root folder as usual.

**Comment by asadm** [comment]
Although not documented (I think), this should already work. Doing `zero src` or `zero www` or even `zero build www` should work just fine. Give it a shot please and see if you find it good enough.

**Comment by iagobruno** [comment]
I searched in the docs before open this issue but did not find this information 😓.
It would be relevant to show this. Wait a minute...

**Comment by asadm** [comment]
@iagobruno Did it work fine for your project?

**Comment by iagobruno** [comment]
Yes, much better than [the other solution](https://github.com/remoteinterview/zero/issues/64#issuecomment-512518724) I used in another project.
